src/pixiewps.c

```diff
@@ -293,19 +293,23 @@ unsigned int hardware_concurrency()
 #endif
 }
 
-int find_rtl_es(struct global *wps, char* pin)
+static int find_rtl_es_dir(struct global *wps, char* pin, int dir)
```

Comment:
The asterisk should be next to the variable, not next to the type (I missed this in your previous commit):
```C
char *pin
```